src/lib/cache/subCache.cpp

```diff
@@ -769,6 +771,8 @@ void subCacheItemInsert
   cSubP->servicePath           = strdup(servicePath);
   cSubP->subscriptionId        = strdup(subscriptionId);
   cSubP->expirationTime        = expirationTime;
+  cSubP->failsCounter          = 0;
```

Comment:
Why don't use the failCounter parameter (introduced in L739) instead of 0?

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/0d976f5d55b682e27c153ecd5662ef408f351a1e